treenode.go

```diff
@@ -679,6 +681,9 @@ func (n *TreeNodeInstance) SetConfig(c *GenericConfig) {
 	n.configMut.Lock()
 	defer n.configMut.Unlock()
 	n.config = c
+	for id := range n.sentTo {
+		n.sentTo[id] = false
```

Comment:
I think it should be called only once. One could even argue if we want to enforce that.